nusacrowd/nusa_datasets/wikilingua/wikilingua.py

```diff
@@ -0,0 +1,139 @@
+import os
+import pickle
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from nusantara.utils import schemas
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import Tasks
```

Comment:
Same like comment above, can change this to
```suggestion
from nusacrowd.utils.constants import Tasks
```